cmd/info.go

```diff
@@ -117,42 +118,169 @@ func info(ctx *cli.Context) error {
 		if err != nil {
 			logger.Fatalf("write message: %s", err)
 		}
-		data := make([]byte, 4)
-		n := readControl(f, data)
-		if n == 1 && data[0] == byte(syscall.EINVAL&0xff) {
-			logger.Fatalf("info is not supported, please upgrade and mount again")
+		err = f.Sync()
+		if err != nil {
+			logger.Fatalf("sync control file: %s", err)
+		}
+		var resp vfs.InfoResponse
+		err = resp.Decode(f)
+		_ = f.Close()
+		if err == syscall.EINVAL {
+			legacyInfo(d, path, inode, recursive, raw)
```

Comment:
use `legacyInfo` for the following args.

Reply:
I'm sorry to not understand your change requests. Should I reuse some code?